spring-cloud-aws-samples/spring-cloud-aws-cloud-map-sample/src/main/resources/bootstrap.properties

```diff
@@ -0,0 +1,15 @@
+spring.cloud.aws.cloudmap.region=us-east-1
```

Comment:
Shouldn't these properties just go to `application.properties`? Or is there a benefit of keeping them in `bootstrap.properties` that I am not aware of?